--- origsrc/libbonobo-2.22.0/configure.in	2008-03-10 10:17:20.000000000 -0500
+++ src/libbonobo-2.22.0/configure.in	2008-07-25 20:26:27.515625000 -0500
@@ -31,6 +31,9 @@
   *-*-cygwin*)
     os_win32=no
     SOPREFIX=cyg
+    SEARCHPATHSEP=":"
+    AC_DEFINE(HAVE_DBUS, 1, [whether we have dbus])
+    dbus_requirement="dbus-1 >= 1.0.0 dbus-glib-1 >= 0.74"
     ;;
   *-*-mingw*)
     os_win32=yes
@@ -214,7 +215,7 @@
 		IF_POSIX_PS=#
 	fi
 
-	case "$target_os" in
+	case "$host_os" in
 		cygwin*)
 			IF_BSD_PS=#
 			IF_POSIX_PS=#
--- origsrc/libbonobo-2.22.0/utils/bonobo-slay.in	2007-01-01 17:46:55.000000000 -0600
+++ src/libbonobo-2.22.0/utils/bonobo-slay.in	2008-07-25 20:26:27.515625000 -0500
@@ -60,7 +60,7 @@
 
 @IF_POSIX_PS@ $ps_cmd = "@PS_PATH@ -U $username -opid,args";
 @IF_BSD_PS@   $ps_cmd = "@PS_PATH@ -U $username -xww -opid,command";
-@IF_CYGWIN_PS@   $ps_cmd = "@PS_PATH@ -s -u $username | awk '{print \$1 \" \" \$4}'";
+@IF_CYGWIN_PS@ $ps_cmd = "@PS_PATH@ -s -u $username | awk '{print \$1 \" \" \$NF}'";
 
 # get Bonobo files
 #
